.github/workflows/build.yml

```diff
@@ -105,6 +104,7 @@ jobs:
         shell: pwsh
         run: |
           echo "${PSScriptRoot}/.github/deps/${{ runner.os }}/bin" | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+          state run install-deps-ci
```

Comment:
Can we make this the only change in the file? The change on line 261 makes me nervous.

Reply:
Bah, that's my damn YAML formatter.